modules/RTC/RTC.js

```diff
@@ -559,6 +559,25 @@ export default class RTC extends Listenable {
         return Promise.all(mutePromises);
     }
 
+    /**
+    * Set mute for all local desktop video streams attached to the conference.
+    * @param value The mute value.
+    * @returns {Promise}
+    */
+    setDesktopMute(value) {
+        const mutePromises = [];
+
+        this.getLocalTracks(MediaType.DESKTOP)
+            .forEach(videoTrack => {
+                // this is a Promise
+                mutePromises.push(value ? videoTrack.mute() : videoTrack.unmute());
```

Comment:
I see that you haven't used `setDesktopMute` yet to unmute the track. This won't work for desktop tracks. On mute, desktop tracks are removed from RTCPeerConnection but not from TPC. So they need to be replaced with the new desktop stream on unmute.

Reply:
For this to work, we need to change the _setMuted impl for JitsiLocalTrack. We can never force unmute desktop share so we shouldn't allow it here. WDYT?

Reply:
Good eye! Upon closer inspection we _only_ use this family of functions to muting, never for unmuting. So I removed the parameter to make it clear and avoid footguns.